rest/client/rest.go

```diff
@@ -746,6 +746,31 @@ func (c *communicatorImpl) EnqueueItem(ctx context.Context, patchID string) (int
 	return positionResp.Position, nil
 }
 
+func (c *communicatorImpl) GetCommitQueueItemAuthor(ctx context.Context, projectID, item string) (string, error) {
+	info := requestInfo{
+		method:  get,
+		version: apiVersion2,
+		path:    fmt.Sprintf("/commit_queue/%s/%s/author", projectID, item),
```

Comment:
I remember that we removed projectID from routes that had an item ID. Does this need a project ID?

Reply:
Yes it does, but for example the delete item commit queue route also still takes a project ID in this manner.